File: jsprobelms/day6/primecheck.js
// take a numbers and check if prime exits in a given prime
let num=144244
found=false
while(num!=0){
    digit=num%10
    fact=0
    for(i=2;i<digit;i++){
        if(digit%i==0){
            fact++
            break
        }
    }
    if (fact==0){
        found=true
        break
    }
    num=Math.floor(num/10)
}
if(found){
    console.log(true);
    
}
else{
    console.log(false);
    
}

File: pythonproblems/day3/primecheck.py
# take a number and check if prime digit exits in a given number or not

def check(num):
    num1=num
    found=False
    while num!=0:
        last=num%10
        # print(last)
    
        fact=0
        for i in range(2,last):
            if last%i==0:
                fact+=1
                break
        if fact==0:
            found=True
            break
        else:
            found=False

        num=int(num/10)
    if found==True:
        print(num1,"contains prime digits")
    else:
        print(num1,"does not contain prime digits")



check(2546)
check(46)
check(466666)
check(46666)
check(466662)